### PR TITLE
Use JavaScript Standard Style (https://standardjs.com)

### PR DESCRIPTION
**Pull request by danielbayerlein**

### CODE REVIEW

File: __tests__/plugins/helpers.test.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-var env = { helpers: {}, locals: {} };
-require('../../plugins/helpers')(env, function helpers() {});
+var env = { helpers: {}, locals: {} }
```
Comment (chrishelgert) [inline __tests__/plugins/helpers.test.js]: Shoudn't we change it to let/const?

File: __tests__/plugins/helpers.test.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+require('../../plugins/helpers')(env, function helpers () {})
```
Comment (chrishelgert) [inline __tests__/plugins/helpers.test.js]: Arrow functions vs function name()

**Comment by chrishelgert** [comment]
On the otherside we have scripts on the client that wont work on every browser without babel 😐
What do you thing about it?

**Comment by danielbayerlein** [comment]
@chrishelgert I think we can migrate the tests to ES6, but in a separate PR.

**Pull request merged**